apps/portal/controllers/dashboards.jag

```diff
@@ -63,6 +63,11 @@ var fresh;
             return;
         }
 
+        if (user && !urlDomain && user.domain != superDomain) {
```

Comment:
I think u can use brackets for code clearity. Or add some comment about this logic.